static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -70,10 +72,49 @@
     }
 }
 
+.adverts--tone-supported {
+    background: none;
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        border-top: 1px solid $news-main-2;
+
+        @include mq(tablet) {
+            left: calc(50% - 370px);
+            right: calc(50% - 370px);
+        }
+
+        @include mq(desktop) {
+            left: calc(50% - 490px);
+            right: calc(50% - 490px);
+        }
+
+        @include mq(leftCol) {
+            left: calc(50% - 570px);
+            right: calc(50% - 570px);
+        }
+
+        @include mq(wide) {
+            left: calc(50% - 650px);
+            right: calc(50% - 650px);
+        }
+    }
+
+    .adverts__logo {
+        color: $guardian-brand-dark;
+    }
+
+    .adverts__row > * + *::before {
+        background: none;
```

Comment:
Removing the separator. It's not clear, I should `content: none` instead and document this line.